### ISSUE TITLE
Release `3.0.0-alpha.1`

### ISSUE THREAD

**Issue by josecelano** [torrust/torrust-index-types-lib#16]
Relates to: https://github.com/torrust/torrust-index-api-lib/issues/40

We want to publish the version `3.0.0` for all Torrust packages at the same time:

- https://github.com/torrust/torrust-tracker
- https://github.com/torrust/torrust-index
- https://github.com/torrust/torrust-index-gui
- https://github.com/torrust/torrust-index-api-lib
- https://github.com/torrust/torrust-index-types-lib

So we should use `3.0.0-alpha.1` not `1.0.0-alpha.4`

cc @da2ce7 @mario-nt

**Comment by mario-nt** [comment]
@josecelano 

I see the point of keeping the same version for all different packages, but I think it doesn't really reflect the real development of the project.

We can have a certain version for the three main repos, and a different version number for the API and types packages.

We are constantly making changes to the tracker, index and gui, so the version number is increased rapidly, while the changes to the API and types libraries are less common, and most of the changes made to the tracker/index/gui don't require updating the API and types libs.

To summarize, having the same version for all the packages would give a cohesive look, but I would rather have the API and types lib with a version that truly reflects the state and changes of those libraries, even if that means having a smaller or bigger version number than the tracker/index/gui.

But If we finally decide to have the same version for all repos, it is still a good idea, as having that cohesive look is also important.

**Comment by josecelano** [comment]
> @josecelano 
> 
> I see the point of keeping the same version for all different packages, but I think it doesn't really reflect the real development of the project.
> 
> We can have a certain version for the three main repos, and a different version number for the API and types packages.
> 
> We are constantly making changes to the tracker, index and gui, so the version number is increased rapidly, while the changes to the API and types libraries are less common, and most of the changes made to the tracker/index/gui don't require updating the API and types libs.
> 
> To summarize, having the same version for all the packages would give a cohesive look, but I would rather have the API and types lib with a version that truly reflects the state and changes of those libraries, even if that means having a smaller or bigger version number than the tracker/index/gui.
> 
> But If we finally decide to have the same version for all repos, it is still a good idea, as having that cohesive look is also important.

Hi @mario-nt I think @da2ce7 's idea is only to use the same version for the major version 3.0.0, after that they can evolve independently and we will need to define dependencies between them using their versions.

**Comment by josecelano** [comment]
![image](https://github.com/torrust/torrust-index-types-lib/assets/58816/4e737aee-555c-4515-994a-cad5af517ad7)

**Comment by josecelano** [comment]
Done: https://www.npmjs.com/package/torrust-index-types-lib/v/3.0.0-alpha.1
